src/metaschema/oscal_control-common_metaschema.xml

```diff
@@ -77,7 +77,9 @@
                 </model>
                 <constraint>
                         <allowed-values target="prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal')]/@name">
-            &allowed-values-control-group-property-name;
+                                &allowed-values-control-group-property-name;
+                                <enum xmlns="http://csrc.nist.gov/ns/oscal/metaschema/1.0" value="method-id">its an assement method.</enum>
```

Comment:
This constraint has poor description AND adds an allowed value "method-id" which is proposed to be removed from the  assessment layer.